src/app-nordigen/app-nordigen.js

```diff
@@ -10,6 +10,18 @@ import { handleError } from './util/handle-error.js';
 import validateUser from '../util/validate-user.js';
 
 const app = express();
+app.get('/link', function (req, res) {
+  res.send(`
+<script>window.close();</script>
```

Comment:
Small nit: Ideally this would be a valid HTML document (including `<!DOCTYPE html>` so we don’t end up with quirks mode weirdness).

Reply:
Makes sense: https://github.com/actualbudget/actual-server/pull/160/commits/eb528cacfcd11f6287eb41c8ebc27b968d0e5d74